diff --git a/node_modules/@convex-dev/auth/dist/server/provider_utils.js b/node_modules/@convex-dev/auth/dist/server/provider_utils.js
index 6bf9673..1c97838 100644
--- a/node_modules/@convex-dev/auth/dist/server/provider_utils.js
+++ b/node_modules/@convex-dev/auth/dist/server/provider_utils.js
@@ -43,13 +43,13 @@ export function configDefaults(config_) {
  */
 export function materializeProvider(provider) {
     const config = { providers: [provider] };
-    materializeAndDefaultProviders(config);
-    return config.providers[0];
+    const result = materializeAndDefaultProviders(config);
+    return result.providers[0];
 }
 function materializeProviders(providers) {
     const config = { providers };
-    materializeAndDefaultProviders(config);
-    return config.providers;
+    const result = materializeAndDefaultProviders(config);
+    return result.providers;
 }
 function materializeAndDefaultProviders(config_) {
     // Have to materialize first so that the correct env variables are used
